### PR TITLE
Add initial scaffolding for the Color scheme target.

### PR DESCRIPTION
**Pull request by jverkoey**
Creates a new Color target in the schemes sub-directory. Includes:

- A CocoaPods target.
- A BUILD file.
- A barebones readme.
- An empty umbrella header.

Pivotal story: https://www.pivotaltracker.com/story/show/156357414

### CODE REVIEW

**Comment by jverkoey** [comment]
Failures are unrelated test failures in Ink. #3173

File: MaterialComponents.podspec [hunk 1]
```diff
@@ -37,6 +37,16 @@ Pod::Spec.new do |mdc|
   #  end
   #
 
+  mdc.subspec "schemes" do |scheme_spec|
+
```
Comment (romoore) [inline MaterialComponents.podspec]: Nit?: Remove extra newline? One more below.

File: components/schemes/Color/BUILD [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//:material_components_ios.bzl",
+     "mdc_public_objc_library",
```
Comment (romoore) [inline components/schemes/Color/BUILD]: Nit: Order of imports

File: MaterialComponents.podspec [hunk 3]
```diff
@@ -37,6 +37,16 @@ Pod::Spec.new do |mdc|
   #  end
   #
 
+  mdc.subspec "schemes" do |scheme_spec|
```
Comment (romoore) [inline MaterialComponents.podspec]: Why not have 

MaterialComponents/Color
MaterialComponents/Motion
MaterialComponents/Shape
MaterialComponents/Typography

Why do we want a /scheme/* ?

File: components/schemes/Color/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<!--docs:
```
Comment (romoore) [inline components/schemes/Color/README.md:1]: Needs copyright stanza?

**Review by romoore — CHANGES_REQUESTED** [review]

File: MaterialComponents.podspec (hunk 1 above)
Comment (jverkoey) [reply MaterialComponents.podspec]: Moved down beside private. Spacing matches the existing style.

File: components/schemes/Color/BUILD (hunk 2 above)
Comment (jverkoey) [reply components/schemes/Color/BUILD]: Done.

File: MaterialComponents.podspec (hunk 3 above)
Comment (jverkoey) [reply MaterialComponents.podspec]: They aren't and shouldn't be considered as components.

File: components/schemes/Color/README.md (hunk 4 above)
Comment (jverkoey) [reply components/schemes/Color/README.md:1]: Not in README's.

Comment (romoore) [reply components/schemes/Color/README.md:1]: Do we know the copyright status of the text written within a README if it's not covered by the same license as the prose in our source (comments)?

Comment (jverkoey) [reply components/schemes/Color/README.md:1]: This is out of scope for this PR, but worth discussing.

**Review by romoore — APPROVED** [review]

**Pull request merged**